be/src/runtime/lake_tablets_channel.cpp

```diff
@@ -537,7 +537,7 @@ Status LakeTabletsChannel::_create_delta_writers(const PTabletWriterOpenRequest&
         std::sort(tablet_ids.begin(), tablet_ids.end());
         DCHECK_EQ(_delta_writers.size(), tablet_ids.size());
         for (size_t i = 0; i < tablet_ids.size(); ++i) {
-            _tablet_id_to_sorted_indexes.emplace(tablet_ids[i], i);
```

Comment:
What's the difference?

Reply:
Please add some comments, and would be better if you can add some unit tests.

Reply:
i will add test case in other pr

Reply:
add some comments?

Reply:
> emplace will fail when key is exists, and i will add test case in other pr

I know this difference, but I don't know how this can introduce correctness issue.

Reply:
> > emplace will fail when key is exists, and i will add test case in other pr
> 
> I know this difference, but I don't know how this can introduce correctness issue.

Add some comments to explain that the key may exist and if so, overwrite the value.